RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -630,6 +645,18 @@ void consumeStuff()
 		borisDemandSandwich(true);
 	}
 
+	// guilty sprouts provide big stats
+	if(auto_is_valid($item[guilty sprout]) && canEat($item[guilty sprout]) && my_level() < 13)
+	{
+		// important for leveling. Attempt to pull if we don't have one
+		pullXWhenHaveY($item[guilty sprout], 1, 0);
+
+		// use food to level if ready for it
+		if(prepare_food_xp_multi())
```

Comment:
prepare_food_xp_multi() was only used in plumber before, it pulls Special Seasoning but maybe that only made sense in plumber, 5-10 is a very small bonus without Ready to Eat

I don't think prepare_food_xp_multi() will make this wait for red rocket if you have VIP but have Everything Looks Red

Reply:
Agree on special seasoning. Added check to only pull if we have the ready to eat effect.

Good find on not waiting. Removed check for everything looks red. I think it should wait now if you have fireworks shop. Will test today